clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/ChangelogClientConfig.java

```diff
@@ -25,7 +25,12 @@ public class ChangelogClientConfig<T extends SpecificRecord> {
    * This will be used in BootstrappingVeniceChangelogConsumer to determine when to sync updates with the underlying
    * storage engine, e.g. flushes entity and offset data to disk. Default is 32 MB.
    */
-  private long databaseSyncBytesInterval = 33554432;
+  private long databaseSyncBytesInterval = 32 * 1024 * 1024L;
+
+  /**
+   * RocksDB block cache size per BootstrappingVeniceChangelogConsumer. Default is 16 GB.
+   */
+  private long rocksDBBlockCacheSizeInBytes = 16 * 1024 * 1024 * 1024L;
```

Comment:
Let's.... set this to something kind of small?  Maybe even nonexistent.  The block cache for bootstrapping consumer is practically never used.  During consumption we only write to rocksdb, and when we read the rocksdb, it'll be after a restart, so the block cache will be cold.

In fact.  I wonder if we shouldn't just turn it off